src/core/compat/zygote.jl

```diff
@@ -0,0 +1,39 @@
+struct ZygoteAD <: ADBackend end
+ADBackend(::Val{:zygote}) = ZygoteAD
+function setadbackend(::Val{:zygote})
+    ADBACKEND[] = :zygote
+end
+
+"""
+gradient_logp_reverse(
+    θ::AbstractVector{<:Real},
+    vi::VarInfo,
+    model::Model,
+    sampler::AbstractSampler=SampleFromPrior(),
+    backend::ADBackend,
+)
+
+Computes the value of the log joint of `θ` and its gradient for the model
+specified by `(vi, sampler, model)` using reverse-mode AD from the specified `backend`, e.g. `TrackerAD()` which uses `Tracker.jl` or `ZygoteAD()` which uses `Zygote.jl`.
+"""
```

Comment:
The function signature in the docstring is different from the one that's implemented. I'm wondering if it's actually useful/needed to copy-paste the docstring from the Tracker implementation here?

Reply:
You are right, I will remove it.